plugins/scsi/fu-scsi-device.c

```diff
@@ -173,6 +173,14 @@ fu_scsi_device_probe(FuDevice *device, GError **error)
 		}
 	}
 
+	/* fake something as we cannot use ioctls */
+	if (fu_device_has_private_flag(device, FU_DEVICE_PRIVATE_FLAG_IS_FAKE)) {
+		fu_device_add_instance_str(device, "VEN", "ACME");
```

Comment:
what would you think about using `FWUPD`, `LINUX` or `TEST` for the fake vendor ID instead?  Just don't want to some day run into a problem that we have a "real" ACME some day.

Reply:
100% valid -- fixed, thanks.